### ISSUE TITLE
Flickr30 Annotations

### ISSUE THREAD

**Issue by chs20** [mlfoundations/open_flamingo#190]
Hi, could you share the Flickr30 annotations json file that you are using, or how to obtain it? Thank you!

**Comment by anas-awadalla** [comment]
Yes this is the [version](https://drive.google.com/file/d/1h5UCKJ_HXErmO0btLneWItKwxBwFpcy8/view?usp=share_link) we use.
